src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -598,6 +614,14 @@ public String getQuestionTypeChoiceOption() {
                 if (otherEnabled && mcqOtherWeight < 0) {
                     errors.add(Const.FeedbackQuestion.MCQ_ERROR_INVALID_WEIGHT);
                 }
+
+                // If one of the mcq has a weight assigned but the mcq text is empty trigger this error
```

Comment:
What's the reason for this? If an empty MCQ string is allowed, there shouldn't be any reason to prevent assigning weight to it?

Reply:
True. Will revert this